### ISSUE TITLE
Conda environment setup errors

### ISSUE THREAD

**Issue by kiwi-sherbet** [zju3dv/NeuralRecon#65]
Hi, I encounter issues when I try to setup the condo environment. The error messages I got is attached below.
Can I ask you for advice to solve the problem?

Thank you!

```
Collecting package metadata (repodata.json): done
Solving environment: done
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
Installing pip dependencies: / Ran pip subprocess with arguments:
['/home/specfazhou/anaconda3/envs/mingyo/bin/python', '-m', 'pip', 'install', '-U', '-r', '/home/specfazhou/neural-mingyo/condaenv.loa284ev.requirements.txt']
Pip subprocess output:

Pip subprocess error:
ERROR: Exception:
Traceback (most recent call last):
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_internal/cli/base_command.py", line 173, in _main
    status = self.run(options, args)
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_internal/cli/req_command.py", line 203, in wrapper
    return func(self, options, args)
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_internal/commands/install.py", line 286, in run
    reqs = self.get_requirements(args, options, finder, session)
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_internal/cli/req_command.py", line 385, in get_requirements
    filename, finder=finder, options=options, session=session
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_internal/req/req_file.py", line 135, in parse_requirements
    for parsed_line in parser.parse(filename, constraint):
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_internal/req/req_file.py", line 319, in parse
    yield from self._parse_and_recurse(filename, constraint)
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_internal/req/req_file.py", line 348, in _parse_and_recurse
    yield from self._parse_and_recurse(req_path, nested_constraint)
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_internal/req/req_file.py", line 324, in _parse_and_recurse
    for line in self._parse_file(filename, constraint):
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_internal/req/req_file.py", line 353, in _parse_file
    _, content = get_file_content(filename, self._session)
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_internal/req/req_file.py", line 518, in get_file_content
    resp = session.get(url)
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_vendor/requests/sessions.py", line 555, in get
    return self.request('GET', url, **kwargs)
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_internal/network/session.py", line 454, in request
    return super().request(method, url, *args, **kwargs)
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_vendor/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_vendor/requests/sessions.py", line 697, in send
    r.content
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_vendor/requests/models.py", line 836, in content
    self._content = b''.join(self.iter_content(CONTENT_CHUNK_SIZE)) or b''
  File "/home/specfazhou/anaconda3/envs/mingyo/lib/python3.7/site-packages/pip/_vendor/requests/models.py", line 769, in generate
    chunk = self.raw.read(chunk_size)
AttributeError: 'FileNotFoundError' object has no attribute 'read'

failed

CondaEnvException: Pip failed
```

**Comment by ngoclinhle** [comment]
edit environment.yaml:
`-    - -r file:requirements.txt
+    - -r requirements.txt`

**Comment by ngoclinhle** [comment]
edit environment.yaml
```
-    - -r file:requirements.txt
+    - -r requirements.txt
```

**Comment by JiamingSuen** [comment]
Fixed in cd047e2.
